addons/block_code/ui/blocks/block/block.gd

```diff
@@ -127,9 +127,9 @@ func update_resources(undo_redo: EditorUndoRedoManager):
 
 	var serialized_props = get_serialized_props()
 
-	if serialized_props != resource.serialized_block.serialized_props:
-		undo_redo.add_undo_property(resource.serialized_block, "serialized_props", resource.serialized_block.serialized_props)
-		undo_redo.add_do_property(resource.serialized_block, "serialized_props", serialized_props)
+	if serialized_props != resource.block_serialized_properties.serialized_props:
+		undo_redo.add_undo_property(resource.serialized_block, "serialized_props", resource.block_serialized_properties.serialized_props)
```

Comment:
This is generating an error because the property name is outdated still I think.
```suggestion
		undo_redo.add_undo_property(resource.block_serialized_properties, "serialized_props", resource.block_serialized_properties.serialized_props)
```

Reply:
Good catch @wnbaum ! This is the value of code reviews.